src/Components/Filters_view/Filters_view.jsx

```diff
@@ -0,0 +1,228 @@
+import { useFilter } from "../../context/FilterContext";
+
+export const Filters_view = () => {
+  const { filter_state, setFilterState } = useFilter();
+  return (
+    <div className="column side-bar pad-1-5 pad-t-3 pad-b-5">
+      <div className="flex jst-sp-sb ali-ce">
+        <p className="fnt-1-2 fnt-w-800">Filters</p>
+        <a
+          onClick={() => {
+            setFilterState({ type: "clear_filter" });
+          }}
+          className="clear-filter"
+        >
+          <p>Clear All</p>
+          <div className="very-thin-bor hundred-w"></div>
+        </a>
+      </div>
+      <div className="spacer"></div>
+      <div className="flex-center-column ali-st gap-1">
+        <label className="fnt-1-2">
+          <p>
+            Price <span className="fnt-w-600">1000</span> to{" "}
+            <span className="fnt-w-600">{filter_state.price}</span>
+          </p>
+          <input
+            type="range"
+            min="1000"
+            max="30000"
+            value={filter_state.price}
+            step="100"
+            className="slider cursor-pointer"
+            onChange={(e) => {
+              setFilterState({ type: "price", paylod: e.target.value });
+            }}
+          />
+        </label>
+      </div>
+      <div className="spacer"></div>
+      <div className="flex-center-column ali-st gap-0-8">
+        <div className="fnt-1 fnt-w-600">Category</div>
+        <label
+          className="custom-checkbox flex-center-row gap-0-8"
+          tab-index="0"
+          aria-label="Checkbox Label"
+          onChange={() => {
+            setFilterState({ type: "category", paylod: "guitar" });
+          }}
+        >
+          <input
+            type="checkbox"
+            checked={filter_state.category.includes("guitar")}
+          />
+          <span className="checkmark"></span>
+          <span className="label fnt-1-2">Guitar</span>
+        </label>
+
+        <label
+          className="custom-checkbox flex-center-row gap-0-8"
+          tab-index="0"
+          aria-label="Another Label"
+          onChange={() => {
+            setFilterState({ type: "category", paylod: "drum" });
+          }}
```

Comment:
you can just make a categories array and loop over it using map instead of writing label and input tags repeatedly